engine/src/main/scala/cromwell/engine/workflow/WorkflowStoreActor.scala

```diff
@@ -2,19 +2,19 @@ package cromwell.engine.workflow
 
 import java.time.OffsetDateTime
 
-import akka.actor.{ActorLogging, Actor, Props}
-import akka.event.Logging
+import akka.actor.{ActorLogging, LoggingFSM, Props}
 import cromwell.core.{WorkflowId, WorkflowSourceFiles}
 import cromwell.database.obj.WorkflowMetadataKeys
-import cromwell.engine.workflow.WorkflowStore.WorkflowToStart
-import cromwell.engine.workflow.WorkflowStoreActor._
-import cromwell.services.MetadataServiceActor.PutMetadataAction
+import cromwell.engine.workflow.WorkflowStoreDatabase.WorkflowToStart
+import cromwell.engine.workflow.WorkflowStoreActor.{WorkflowsBatchSubmittedToStore, _}
+import cromwell.services.MetadataServiceActor.{MetadataPutAcknowledgement, PutMetadataAction}
 import cromwell.services.{MetadataEvent, MetadataKey, MetadataValue, ServiceRegistryClient}
 
+import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
 import scalaz.NonEmptyList
 
-object WorkflowStore {
+object WorkflowStoreDatabase {
```

Comment:
Same comment as above. 

One could argue that database is being used generically and not to mean an RDBMS but I'd counter that if that's the case then store and database are synonymous here